README.md

```diff
@@ -6,15 +6,19 @@ This repo contains the code for deploying, populating and interacting with a Tru
 
 The easiest way to test/exercise this code is by creating a login.txt file (see below) and then launching `server.py`, and then using the snapshot tests to exercise the API and its underlying database.
 
-### Create login.txt file
+### Update trusat-config.yaml file
 
 This should be a text file containing the parameters of the `Database` contructor - with one parameter on each line. At the time of writing that means:
 
-    dbname     - name of the database
-    dbtype     - database type: "INFILE", "sqlserver" or "sqlite3" (without quotes)
-    dbhostname - hostname for sqlserver
-    dbusername - username for sqlserver
-    dbpassword - password for sqlserver
+    # Trusat Database Connection Configuration
+    Database:
+     name: "space"
+     type:
+     hostname: "127.0.0.1"
+     username: "root"
+     password:
+
+Place the updated file in the parent directory of ./trusat-orbit to avoid commiting your sensitive data to GIT.
```

Comment:
A common practice is to tell the user how to configure this file but not supply it in order to add it to the git ignore file. This works fine, just a matter of flavors in case this alternative is preferred in the long run.